browser/ui/android/strings/android_chrome_strings.grd

```diff
@@ -884,7 +884,7 @@ Your Brave account may have other forms of browsing history like searches and ac
         Safety check <ph name="BEGIN_NEW">&lt;new&gt;</ph>New<ph name="END_NEW">&lt;/new&gt;</ph>
       </message>
       <message name="IDS_SAFETY_CHECK_DESCRIPTION" desc="Introduces the safety check categories to the user. Followed by several items, such as 'Safe Browsing', 'Updates', 'Passwords'.">
-        Brave can help keep you safe from data breaches, unsafe websites, and more
```

Comment:
Interesting. The original wording for Android ("unsafe website") sounds better than what I proposed.

Maybe we should go with "unsafe websites, bad extensions, and more" everywhere instead of "malicious websites, bad extensions, and more"?